### PR TITLE
feat: restore jobs

### PR DESCRIPTION
**Pull request by revolunet**
Add a restore container manual job. database restored is disabled atm

copy containers : `dev/cdtn` to `dev/cdtn-dev`

should we limit gitlab job to branches ?

### CODE REVIEW

**Comment by revolunet** [comment]
@douglasduteil : a voir si la technique du `components/restore` et kosko.toml est correcte

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -98,3 +101,26 @@ Deploy app (dev):
...
+    SOURCE_CONTAINER: "cdtn"
+    SOURCE_SERVER: "dev"
+    DESTINATION_CONTAINER: "cdtn-dev"
+    DESTINATION_SERVER: "dev"
+    # db restore
+    # BACKUP_DB_NAME: "db_${CI_COMMIR_SHORT_SHA}"
+    # BACKUP_DB_OWNER: "user_${CI_COMMIR_SHORT_SHA}"
+    # BACKUP_DB_FILE: "hasura_prod_db.psql.gz"
+    # kosko options
+    KOSKO_GENERATE_ARGS: --env dev restore
```
Comment (lionelB) [inline .gitlab-ci.yml:126]:
```suggestion
    # only generate restore job
    KOSKO_GENERATE_ARGS: --env restore
```

Comment (revolunet) [reply .gitlab-ci.yml]: i set --env prod. (env is mandatory)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -98,3 +101,26 @@ Deploy app (dev):
 Deploy app (prod):
   only:
     - never
+
+#
+
+# todo: restrict to branches ?
+Restore container:
+  stage: .post
```
Comment (lionelB) [inline .gitlab-ci.yml:109]:
```suggestion
  stage: Deploy
  only:
    - branches
```

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -98,3 +101,28 @@ Deploy app (dev):
...
     - never
+
+#
+
+# todo: restrict to branches ?
+Restore container:
+  stage: Deploy
+  only:
+    - branches
+  when: manual
```
Comment (lionelB) [inline .gitlab-ci.yml:112]:
```suggestion

```

**Pull request merged**